.github/workflows/main.yml

```
@@ -69,112 +74,193 @@
+
+            echo "Backend rolled back to image: $BACKEND_ROLLED_IMAGE"
+            echo "Frontend rolled back to image: $FRONTEND_ROLLED_IMAGE"
           EOF
+
+  cleanup:
+    needs: [deploy, rollback]
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)